### ISSUE TITLE
When using custom hyperkube image it is not used for kube-proxy

### ISSUE THREAD

**Issue by jchauncey** [Azure/aks-engine#3013]
I built a custom hyperkube cluster using 0.47.0 and noticed that the kube-proxy running on my master node was not using the custom hyperkube image I was testing.

```shell
azureuser@k8s-master-21703565-0:~$ docker ps | grep hyperkube
7c604053d528        k8s.gcr.io/hyperkube-amd64                             "/hyperkube kube-pro…"   2 hours ago         Up 2 hours                              k8s_kube-proxy_kube-proxy-hxd5w_kube-system_c1732cc8-4ccd-4e5d-b56c-92792fa77f95_0
8fbe76d47774        1b4176fb5f39                                           "/hyperkube kube-api…"   2 hours ago         Up 2 hours                              k8s_kube-apiserver_kube-apiserver-k8s-master-21703565-0_kube-system_4a049324fe65112f85eda19d5c16c89b_0
403fa3bb8748        1b4176fb5f39                                           "/hyperkube kube-sch…"   2 hours ago         Up 2 hours                              k8s_kube-scheduler_kube-scheduler-k8s-master-21703565-0_kube-system_d8fb2556635a5b34b526c2614257df03_0
6632b72214dc        1b4176fb5f39                                           "/hyperkube kube-con…"   2 hours ago         Up 2 hours                              k8s_kube-controller-manager_kube-controller-manager-k8s-master-21703565-0_kube-system_0054ddd9bd8832214aa15a1df16a33a6_0
```

The image id 1b4 is my custom image:
```shell
azureuser@k8s-master-21703565-0:~$ docker images | grep 1b4
upstream.azurecr.io/oss/kubernetes/hyperkube           v1.15.10-hotfix.20200326   1b4176fb5f39        7 days ago          643MB
```

I expected that the hyperkube image would be used in all places but that doesnt seem to be the case.

**Comment by chewong** [comment]
This can be fixed by adding:

```json
...
"customKubeProxyImage": "<custom hyperkube image>",
"customHyperkubeImage": "<custom hyperkube image>",
...
```

in the API model.

**Comment by chewong** [comment]
FIxed by #3538
